connectors/sources/postgresql.py

```diff
@@ -218,6 +219,9 @@ async def ping(self):
     async def get_tables_to_fetch(self, is_filtering=False):
         tables = configured_tables(self.tables)
         if is_wildcard(tables) or is_filtering:
+            self._logger.info(
```

Comment:
Same reasoning as in `mssql.py`: I think we should've two distinct log messages instead of using one containing `or`.